src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java

```diff
@@ -145,7 +144,7 @@ public void generateRecommendation(KruizeObject kruizeObject, List<ExperimentRes
 
                         // Check for min data before setting notifications
                         // Check for at least short term
-                        if (!RecommendationUtils.checkIfMinDataAvailableForTerm(containerDataKruizeObject, RecommendationConstants.RecommendationTerms.SHORT_TERM)) {
+                        if (!RecommendationUtils.checkIfMinDataAvailableForTerm(containerDataKruizeObject, RecommendationConstants.RecommendationTerms.SHORT_TERM, monitoringEndTime)) {
```

Comment:
It appears that this code is checking only for `SHORT_TERM` and if it is not available then setting the notification `INFO_NOT_ENOUGH_DATA`

Reply:
Fixed now. Added test for the same as well.